src/components/ec-modal/ec-modal.spec.js

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable no-use-before-define */
+import { mount } from '@vue/test-utils';
+import EcModal from './ec-modal.vue';
+
+describe('EcModal', () => {
+  it('should not render the modal', () => {
+    const wrapper = mount(EcModal);
+    expect(wrapper.find('.ec-modal').exists()).toBe(false);
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it('should render basic modal', () => {
+    const wrapper = mount(EcModal, {
+      propsData: {
+        showModal: true,
+      },
+      slots: {
+        main: '<p>Before we can process your application we need you to upload your management accounts. You can do this now or leave it for later.</p>',
+      },
+    });
+    expect(wrapper.element).toMatchSnapshot();
+  });
+  it('should render close button', () => {
+    const wrapper = mount(EcModal, {
+      propsData: {
+        showModal: true,
+      },
+    });
+    expect(wrapper.find('.ec-modal__close').exists()).toBe(true);
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it('should not render close button if set to false', () => {
```

Comment:
`should render help link when showCloseIcon is set to false`, just like it is in `should render help link when showHelpLink is set to true`